lib/puppet/provider/aem_crx_package/ruby.rb

```diff
@@ -103,7 +103,46 @@ def build_client
     @client
   end
 
+  def wait_for_install_ok
+    require 'uri'
+    require 'json'
+    require 'net/http'
+    retries ||= @resource[:retries]
+    retry_timeout = @resource[:retry_timeout]
+    host = 'http://localhost:4502'
```

Comment:
The fixed port '4502' should come from the "port" parameter -> see build_cfg method

Reply:
True, but for that i needed to modify the 'build_client' function, which i did not want to do for this PR. It is mainly intended as a discussion entry-point and definitely not as a final clean-code PR.